### ISSUE TITLE
PouchDB test broken: Syncing should stop

### ISSUE THREAD

**Issue by nolanlawson** [pouchdb/express-pouchdb#255]
As soon as we fix one issue, another one pops up. :stuck_out_tongue: 

```
  1) test.sync.js-http-http Syncing should stop if one replication fails (issue 838):
     Error: timeout of 50000ms exceeded
      at null.<anonymous> (/home/travis/build/pouchdb/pouchdb/node_modules/mocha/lib/runnable.js:158:19)
      at Timer.listOnTimeout [as ontimeout] (timers.js:121:15)
```

e.g. https://travis-ci.org/pouchdb/pouchdb/jobs/78987263. caused by https://github.com/pouchdb/pouchdb/pull/4146

**Comment by nolanlawson** [comment]
Hm, can't reproduce.

**Comment by nolanlawson** [comment]
I have tested this a half-dozen times on both Linux and Mac, and I can't reproduce. :( Seems consistent in Travis, though.

**Comment by nolanlawson** [comment]
Aha! The bug only exists in Node 0.10. I can repro.

**Comment by nolanlawson** [comment]
Bluebird logs the error:

```
{ [Error: ETIMEDOUT] code: 'ETIMEDOUT', connect: null, status: 400 }
Error: ETIMEDOUT
    at null._onTimeout (/Users/nolan/workspace/pouchdb/node_modules/request/request.js:818:15)
    at Timer.listOnTimeout [as ontimeout] (timers.js:121:15)
Unhandled rejection Error: ETIMEDOUT
    at onError (/Users/nolan/workspace/pouchdb/lib/deps/ajax/ajaxCore.js:61:18)
    at Request._callback (/Users/nolan/workspace/pouchdb/lib/deps/ajax/ajaxCore.js:106:14)
    at self.callback (/Users/nolan/workspace/pouchdb/node_modules/request/request.js:198:22)
    at Request.emit (events.js:95:17)
    at null._onTimeout (/Users/nolan/workspace/pouchdb/node_modules/request/request.js:821:12)
    at Timer.listOnTimeout [as ontimeout] (timers.js:121:15)
    1) Syncing should stop if one replication fails (issue 838)
```

**Comment by nolanlawson** [comment]
"Fixed" by upgrading to 0.12 in our Travis tests. ¯\_(ツ)_/¯

**Comment by marten-de-vries** [comment]
1808abd
